### PR TITLE
Add utility function serialization.serialize_segmentation

### PR DESCRIPTION
**Pull request by raphaelmerx**
## Description

* Adds a function `serialize_segmentation` to save the result of `blla.segment` to e.g. ALTO XML
* Add an example usage of this function in the "API Quickstart", and the reference to the API reference
* Add a test for this new function 
* Make use of it in `kraken segmenter`

## Demo

```python

# load a segmentation model

with PIL.Image.open('foo.jpg') as im:
    res = blla.segment(im, model=model)
    size = im.size
with open('foo.xml', 'w') as fp:
    fp.write(serialization.serialize_segmentation(res, 'foo.jpg', size, template='alto'))
```

### CODE REVIEW

File: docs/_static/custom.css [hunk 1]
```diff
@@ -19,3 +19,6 @@ div.rightside {
     margin-left: 125px;
 }
 
+dl.py {
```
Comment (raphaelmerx) [inline docs/_static/custom.css:22]: This is just adding some margin between each function in the API reference:

Screenshot 2022-02-22 at 18 06 26

File: tests/test_serialization.py [hunk 2]
```diff
@@ -220,3 +220,12 @@ def test_region_only_pagexml_serialization_validation(self):
 
         fp.write(serialization.serialize([], image_name='foo.png', template='pagexml', regions=self.bl_regions))
         validate_page(self, fp)
+
+    def test_serialize_segementation_alto(self):
```
Comment (mittagessen) [inline tests/test_serialization.py:224]: Could you add a test for PageXML as well? And there's a typo in the function name.

**Comment by mittagessen** [comment]
Seems good. There's a small thing about the test but otherwise it looks immediately mergable.

File: tests/test_serialization.py (hunk 2 above)
Comment (raphaelmerx) [reply tests/test_serialization.py]: Added a test for PageXML and fixed the typo in https://github.com/mittagessen/kraken/pull/329/commits/48d93a606b7b9eaad76518e34e557640dd6a9e7d

File: tests/test_serialization.py [hunk 3]
```diff
@@ -48,7 +48,8 @@ def validate_page(self, fp):
 
     ids = [x.get('id') for x in doc.findall('.//*[@id]')]
     counts = Counter(ids)
-    self.assertEqual(counts.most_common(1)[0][1], 1, msg='Duplicate IDs in PageXML output')
+    if len(counts):
```
Comment (raphaelmerx) [inline tests/test_serialization.py:51]: Had to add this check as `doc.findall('.//*[@id]')` is not returning any element in the segmentation test

File: pytest.ini [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[pytest]
```
Comment (raphaelmerx) [inline pytest.ini:1]: Adding this minimal pytest config so code editors can auto-discover tests for this project.

**Comment by mittagessen** [comment]
Thanks.

**Pull request merged**